### PR TITLE
Add touch events to mobile Carousel

### PR DESCRIPTION
**Pull request by atma**

### CODE REVIEW

**Comment by battaglr** [comment]
Sadly this is not working properly on Android 6 (Chrome), at the point that is not usable in the current state. :cry:

*In iOS is working great.*

Any ideas?

File: src/mobile/scripts/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+         * @type {Number}
+         */
+        this._endTouchX = 0;
+
+        /**
+         * The X position of the last touch.
+         * @private
+         * @type {Number}
+         */
+        this._partialTouchX;
```
Comment (battaglr) [inline src/mobile/scripts/Carousel.js]: Is this OK? Don't we need to do an assignment here like: `this._partialTouchX = 0;`?

File: dist/mobile/chico.js [hunk 2]
```diff
@@ -7305,6 +7305,302 @@ ch.factory = function (Klass, fn) {
...
+        } else { // if the touch distance is minnor that minimum touch width, 
+            
+            // back carousel to previus position
+            this._captureMove(-this._pageWidth * (this._currentPage - 1));
+
+        }
+
+        // After touch end, available scroll again
+        if (this._disabledScrollY) {
+            document.body.style.overflow = 'initial';
```
Comment (battaglr) [inline dist/mobile/chico.js]: The `initial` property has a [decent support](http://caniuse.com/#search=initial), but maybe we could use `visible` that's the [default (initial) value](https://developer.mozilla.org/en/docs/Web/CSS/overflow). Thoughts?

File: dist/mobile/chico.js [hunk 3]
```diff
@@ -7305,6 +7305,302 @@ ch.factory = function (Klass, fn) {
...
+    Carousel.prototype._translate = (function () {
+        // CSS property written as string to use on CSS movement
+        var transform = '-' + ch.util.VENDOR_PREFIX + '-transform';
+        
+        // Use CSS transform to move
+        if (ch.support.transition) {
+            return function (displacement) {
+                this._$list.css(transform, 'translate3d(' + displacement + 'px,0,0)');
+                this._displacement = displacement;
+                // console.log('_translate():: displacement', displacement);
```
Comment (battaglr) [inline dist/mobile/chico.js]: Ops! Commented `console.log`. :sweat_smile:

**Comment by atma** [comment]
@battaglr Let´s invite the author @1987santiago to the conversation 😉

**Comment by battaglr** [comment]
@atma, my bad! :expressionless:

**Pull request merged**